bower.json

```diff
@@ -13,6 +13,7 @@
   ],
   "dependencies": {
     "listjs": "~1.1.1",
-    "openseadragon": "~1.2.0"
+    "openseadragon": "~1.2.0",
```

Comment:
Okay, I merged that one first (earlier today). I hope that doesn't cause a problem.